sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -116,8 +116,9 @@ class SessionCatalog(
     currentDb = db
   }
 
-  def getDefaultDBPath(db: String): String = {
-    System.getProperty("java.io.tmpdir") + File.separator + db + ".db"
+  def getDatabasePath(dbName: String, path: Option[String]): String = {
```

Comment:
Actually, I am also wondering if we should return an empty String in non-Hive tables? Let me know if we need to remove it.